substrate/client/transaction-pool/src/lib.rs

```diff
@@ -659,8 +659,13 @@ where
 					})
 					.unwrap_or_default()
 					.into_iter()
-					.filter(|tx| tx.is_signed().unwrap_or(true));
-
+					// TODO [#2415]: This isn't really what we mean - we really want a
```

Comment:
Will this then be an issue for e.g. a Frontier chain? if none of their transactions is propagated it is an issue.

Reply:
The code that is there now, `.filter(|tx| !tx.is_bare())`, will do the same thing as `.filter(|tx| tx.is_signed().unwrap_or(true))`, what am I missing?